fem/bilininteg_mass_pa.cpp

```diff
@@ -41,6 +41,8 @@ void MassIntegrator::SetupPA(const FiniteElementSpace &fes, const bool force)
       InitCeedCoeff(Q, ptr);
       return CeedPAMassAssemble(fes, *ir, *ptr);
    }
+#else
+   MFEM_CONTRACT_VAR(force);
```

Comment:
I don't understand this line, can you tell my why it is needed?

Reply:
it removes one compiler warning when we're not using `libCeed` and compiling with `nvcc`.